translations/deckbuilding.tex/pl.po

```diff
@@ -437,7 +437,7 @@ msgid ""
 "stop playing cards, the power dissipates.  You may use a cheaper "
 "\\svg{empower} effect than what you've paid for, though."
 msgstr ""
-"Karty Zaklęć\\index{Karta!Zaklęcia} mają trzy możliwe efekty podstawowe. "
+"Karty Zaklęć\\index{Karta!Zaklęć} mają trzy możliwe efekty podstawowe. "
```

Comment:
hm.. Efekt Podstawowy nie był pisany wielkimi literami?

Reply:
Hm, mógł być. Teraz to jest niekonsekwentne w oryginale, jak inni ludzie piszą.